spec/forms/idv/api_document_verification_status_form_spec.rb

```diff
@@ -8,12 +8,14 @@
     )
   end
 
-  let(:async_state) { ProofingDocumentCaptureSessionResult.none }
+  let(:async_state) { DocumentCaptureSessionAsyncResult.new }
```

Comment:
should we name this to like `async_result` ?

Reply:
The name comes from https://github.com/18F/identity-idp/pull/4495/files#diff-5b5622a6a91095f2979ead4030e9b80c16c4b6cac4079a97624d652985e4ba36L6, and my feel is to have them match? I don't feel super strongly

Reply:
oh good point, seems like not worth the change